web/embed-new.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
Here, and for the other new files in `web/` consider moving them to their own, new directory in `web/`. We can also have `new_embed.dart` (below) in the same directory, instead of in a separate scripts dir.

Reply:
Created an `experimental` directory, and flattened all the new files into it.